api/src/Entity/Checklist.php

```diff
@@ -29,28 +29,28 @@
 #[ApiResource(
     operations: [
         new Get(
-            security: 'is_granted("CAMP_COLLABORATOR", object) or is_granted("CAMP_IS_PROTOTYPE", object)'
+            security: 'is_granted("CHECKLIST_IS_PROTOTYPE", object) or is_granted("CAMP_IS_PROTOTYPE", object) or (null != object.getCamp() and is_granted("CAMP_COLLABORATOR", object))'
         ),
         new Patch(
-            security: 'is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)'
+            security: 'is_granted("ROLE_ADMIN") or (null != object.getCamp() and (is_granted("CAMP_MEMBER", object) or is_granted("CAMP_MANAGER", object)))'
```

Comment:
This would allow an admin to patch any checklist by mistake. Shouldn't this be:
```
(null === object.getCamp() and is_granted("ROLE_ADMIN"))
```

Same applies for delete and post operation as well.